it/uk/gov/hmrc/apprenticeshiplevy/config/IntegrationTestConfig.scala

```diff
@@ -45,9 +45,8 @@ trait IntegrationTestConfig {
         "appName" -> "application-name",
         "appUrl" -> "http://microservice-name",
         "microservice.metrics.graphite.enabled" -> "false",
-        "Test.microservice.services.service-locator.host" -> stubHost,
-        "Test.microservice.services.service-locator.port" -> stubPort,
         "microservice.services.service-locator.host" -> stubHost,
+        "microservice.services.service-locator.protocol" -> "http",
```

Comment:
Should this not be 'https'?
```suggestion
        "microservice.services.service-locator.protocol" -> "https",
```